topic-operator/src/main/java/io/strimzi/operator/topic/Labels.java

```diff
@@ -102,7 +101,10 @@ private static void checkLabelKey(String key) {
      * @param string The string to parse.
      * @return The label predicate
      */
-    public static LabelPredicate fromString(String string) throws IllegalArgumentException {
+    public static Labels fromString(String string) throws IllegalArgumentException {
+        if (string.equals("")) {
```

Comment:
Maybe just a check that it's not `null` and the if it's `null` we use the `EMPTY_MAP` as well?